tests/textlab_api/textlab_api_test.py

```diff
@@ -52,6 +53,113 @@ def update_page_content(
     return (edit_page, admin_client.post(edit_page, edit_page_data))
 
 
+def create_page_content(
+    admin_client: Client,
+    title: str,
+    content: str,
+    hix_ignore: bool = False,
+) -> tuple[int, HttpResponse]:
+
+    new_page = reverse(
+        "new_page",
+        kwargs={
+            "region_slug": "augsburg",
+            "language_slug": "de",
+        },
+    )
+    new_page_data = {
+        "title": title,
+        "content": content,
+        "mirrored_page_region": "",
+        "_ref_node_id": 3,
```

Comment:
It seems like your new test interferes with the older test `test_hix_score_update`, because they seem to share the same page. I am not really sure why this is happening, but maybe we can create a completely new object for that? :thinking:

Reply:
Probably because we are using "Willkommen in Augsburg" for both tests

Reply:
Hi Ulli,
Thank you for your comments! At last I figured out that tests were failing due to hix_score caching: https://github.com/digitalfabrik/integreat-cms/blob/d8f632c86efba03a8b75a4263e29a48f82d3a2ab/integreat_cms/cms/views/utils/hix.py#L39
All I had to do to fix tests is to use unique value for content :-) But it was a long journey to find it out )